tox.ini

```diff
@@ -73,15 +72,15 @@ commands = flake8-rst gensim/ docs/ {posargs}
 
 
 [testenv:compile]
-basepython = python2
+basepython = python3.7
```

Comment:
are you sure about base version? Maybe just `python3`? `python3.7` not really frequently available on machine